README.md

```diff
@@ -85,13 +85,14 @@ If you are unsure whether you need to change that version at a given moment, the
 
 ## Upgrading Dependencies
 
-This repository uses `pip-tools` to manage dependencies, so make sure you've installed it with `pip install pip-tools`. To add or update dependencies, change `requirements.in`, Then run
+This repository uses `uv` to manage dependencies.
+Make your dependency updates to `pyproject.toml` then run:
 
 ```
 make update-requirements
```

Comment:
I don’t think this is necessary anymore. The make command still uses `uv pip compile`. `uv sync` will automatically sync the installed dependencies and lockfile depending on the `pyproject.toml` definition.